orttraining/orttraining/test/training_api/trainer/trainer.cc

```diff
@@ -210,7 +236,7 @@ void RunTraining(const TestRunnerParameters& params) {
         end_to_end_start = std::chrono::high_resolution_clock::now();
       }
 
-      std::vector<Ort::Value> inputs;
+      std::vector<OrtValue*> inputs;
```

Comment:
I did not get why we expect user code to use OrtValue instead of Ort::Value?

Reply:
Because we are moving the use code to use C API